src/components/CreateListingModal.tsx

```diff
@@ -31,14 +32,13 @@ function CreateListingModal(props: Props) {
 
     const didClickListNFTButton = () => {
         const contract = KasuContract();
-
         try {
             contract.listNFT(
                 props.tokenID,
                 props.tokenAddress,
                 formValues.rentalDuration,
                 formValues.interestRate,
-                formValues.collateralRequired
+                ethers.utils.parseEther(formValues.collateralRequired)
```

Comment:
@tedwu13 fyi the rest of the app/smart contract assumes this is in Wei so to make it easier I'm gonna just pass in the wei value

Reply:
yeah makes sense thanks for converting it